types/tuya-panel-kit/index.d.ts

```diff
@@ -3416,7 +3416,7 @@ export interface PickerViewProps extends Omit<PickerIOSProps, 'onValueChange' |
      * @description Value matching value of one of the items
      * @defaultValue undefined
      */
-    selectedValue?: string | number | boolean;
+    selectedValue?: string;
     /**
```

Comment:
Not sure exactly what the best fix here is. The upstream flow-typings were refined to only be "string", but I think that was a case of previous types being incorrect. `tuya-panel-kit` seems hard-locked to RN 0.51.0 though, so having it work with evergreen RN typings is a bit suspicious.